src/Sulu/Bundle/AdminBundle/Controller/AdminController.php

```diff
@@ -340,8 +340,15 @@ public function metadataAction(string $type, string $key, Request $request): Res
         $metadata = $this->metadataProviderRegistry->getMetadataProvider($type)
             ->getMetadata($key, $user->getLocale(), $metadataOptions);
 
+        $context = new Context();
+        $context->addGroup('Default');
+        if (true === \filter_var($metadataOptions['onlyKeys'] ?? 'false', \FILTER_VALIDATE_BOOLEAN)) {
+            $context->addGroup('admin_form_metadata_keys_only');
```

Comment:
A functional test would be good for this part of the AdminController. Else the code looks fine.